### PR TITLE
ADAPT-2551: CLI Tool for cloning Netlify sites

### PR DESCRIPTION
**Pull request by nplowman**

### CODE REVIEW

**Comment by nplowman** [comment]
For testing locally:

- Clone the repo and check out this branch.
- Set up the configuration variables for netlify_token and github_token as described in the README.md file. 
- Run `./bin/run netlify-clone` and follow the prompts.

Note that the account you use to generate the Github access token must have permission to add deploy keys.

**Comment by sherakama** [comment]
This looks neat @nplowman. 

This is a good opportunity for @mattanglin and @adil-innovation-lab to review and give this tool a test. Please have a look at this tool and review the PR. Scope for this work can be found here: https://stanfordits.atlassian.net/browse/ADAPT-2551

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "globby": "^10.0.2"
   },
   "engines": {
-    "node": ">=8.0.0"
+    "node": ">=15.0.0"
```
Comment (mattanglin) [inline package.json:17]: Is there a particular reason we're aligning on Node `>=15.0.0`? Odd numbered node versions don't offer any kind of stability or support and generally just exist to allow community testing/feedback. I would recommend we align to an LTS version.

[Node release patterns](https://nodejs.org/en/about/releases/)

Comment (sherakama) [reply package.json:17]: This is probably us. We set our nvmrc to 15 a while back for support on another project and it has 'stuck'. I suspect we can move along to 16 now.
eg:
https://github.com/SU-SWS/adapt-dev-utils/blob/main/.nvmrc#L1

File: README.md [hunk 2]
```diff
@@ -1,2 +1,67 @@
...
+
+### Storyblok Token
+To obtain this token:
+1. Log into Storyblok
+1. Find and click the My account menu link (Reveal it by clicking the caret icon next to the Storyblok logo in the top-left of the UI.)
+1. Find the "Personal access tokens" section, and click "Generate new token"
+
+Then set it using one of the following methods:
+Environment Variable: STORYBLOK_TOKEN=your-token
+CLI Config: `adapt-utils config-set storyblok_token "your-token"
```
Comment (mattanglin) [inline README.md:67]: ❤️  for 📚

File: README.md [hunk 3]
```diff
@@ -1,2 +1,67 @@
...
+## Configuration
+
+Configuration variables can be set in a few different ways:
+
+1. Environment variables. 
+`GITHUB_TOKEN=YOUR_TOKEN`
+1. The config-set command.
+`adapt-utils config-set github_token "your_token"`
+1. Using an option in the CLI command.
+`adapt-utils netlify-cline --githubToken="yourtoken"`
```
Comment (mattanglin) [inline README.md:31]: ~netlify-cline~ => netlify-cl**o**ne

Comment (nplowman) [reply README.md:31]: Fixed! (Sorry - somehow I completely missed these PR comments while I was sidetracked on other tickets the last couple weeks)

File: package.json (hunk 1 above)
Comment (nplowman) [reply package.json:17]: NPM version has been updated to 16.

**Review by sherakama — APPROVED** [review]
Neat. Thanks for this.

**Pull request merged**